test/e2e/data/rancher/rancher-service-patch.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 kind: Service
 metadata:
   annotations:
-    k8s.ngrok.com/app-protocols: '{"https-internal":"HTTPS","http":"HTTP"}'
```

Comment:
I’m not convinced this change is required. As I mentioned before, this annotations are just mapping to the port names in the service. Additionally in my setup now the ui via RANCHER_HOSTNAME is not accessible anymore.

Reply:
It works for me. But let me change this back and try again.....will report back shortly 😄

Reply:
Using current setup neither AWS nor Azure cluster had no issue with connecting agent to the rancher cluster. Here is a screenshot of fully imported AKS cluster https://github.com/rancher-sandbox/rancher-turtles/pull/133#issuecomment-1743339541

Reply:
That worked with reverting it, thanks @Danil-Grigorev